junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvParserFactory.java

```diff
@@ -65,6 +69,7 @@ private static CsvParserSettings createParserSettings(String delimiter, String l
 		settings.getFormat().setQuoteEscape(quote);
 		settings.setEmptyValue(emptyValue);
 		settings.setAutoConfigurationEnabled(false);
+		settings.setMaxCharsPerColumn(maxCharsPerColumn == 0 ? DEFAULT_MAX_CHARS_PER_COLUMN : maxCharsPerColumn);
```

Comment:
❓ Since both annotations define a default value when is this ever `0`?

Reply:
@cemurat Have you seen my question?

Reply:
**Team Decision**: In line with the previous comment, use the `maxCharsPerColumn` as-is at this point in the code base.

Reply:
Updated